functionaltests/src/test/java/specs/BaseFixture.java

```diff
@@ -50,73 +52,115 @@
-    private static final String TEST_DATA_3_DS_PASSWORD = "TEST_DATA_3_DS_PASSWORD";
-    private static final String TEST_DATA_SW_BANK_TRANSFER_IBAN = "TEST_DATA_SW_BANK_TRANSFER_IBAN";
-    private static final String TEST_DATA_SW_BANK_TRANSFER_BIC = "TEST_DATA_SW_BANK_TRANSFER_BIC";
+    // Environment variables names
+    protected static final String TEST_DATA_CT_PAYONE_INTEGRATION_URL = "TEST_DATA_CT_PAYONE_INTEGRATION_URL";
 
-    private static final String TEST_DATA_PAYONE_MERCHANT_ID = "TEST_DATA_PAYONE_MERCHANT_ID";
-    private static final String TEST_DATA_PAYONE_SUBACC_ID = "TEST_DATA_PAYONE_SUBACC_ID";
-    private static final String TEST_DATA_PAYONE_PORTAL_ID = "TEST_DATA_PAYONE_PORTAL_ID";
-    private static final String TEST_DATA_PAYONE_KEY = "TEST_DATA_PAYONE_KEY";
+    protected static final String TEST_DATA_TENANT_NAME = "TEST_DATA_TENANT_NAME";
+    protected static final String TEST_DATA_VISA_CREDIT_CARD_NO_3_DS = "TEST_DATA_VISA_CREDIT_CARD_NO_3DS";
+    protected static final String TEST_DATA_VISA_CREDIT_CARD_3_DS = "TEST_DATA_VISA_CREDIT_CARD_3DS";
+    protected static final String TEST_DATA_3_DS_PASSWORD = "TEST_DATA_3_DS_PASSWORD";
+    protected static final String TEST_DATA_SW_BANK_TRANSFER_IBAN = "TEST_DATA_SW_BANK_TRANSFER_IBAN";
+    protected static final String TEST_DATA_SW_BANK_TRANSFER_BIC = "TEST_DATA_SW_BANK_TRANSFER_BIC";
+    protected static final String TEST_DATA_SW_BANK_TRANSFER_PIN = "TEST_DATA_SW_BANK_TRANSFER_PIN";
+    protected static final String TEST_DATA_SW_BANK_TRANSFER_TAN = "TEST_DATA_SW_BANK_TRANSFER_TAN";
 
-    private static final Random randomSource = new Random();
-    private BlockingSphereClient ctpClient;
-    private URL ctPayoneIntegrationBaseUrl;
-    private LoadingCache<String, Type> typeCache;
+    protected static final String TEST_DATA_PAYONE_MERCHANT_ID = "TEST_DATA_PAYONE_MERCHANT_ID";
+    protected static final String TEST_DATA_PAYONE_SUBACC_ID = "TEST_DATA_PAYONE_SUBACC_ID";
+    protected static final String TEST_DATA_PAYONE_PORTAL_ID = "TEST_DATA_PAYONE_PORTAL_ID";
+    protected static final String TEST_DATA_PAYONE_KEY = "TEST_DATA_PAYONE_KEY";
 
-    /** maps legible payment names to payment IDs (and vice versa)*/
+    protected static final String TEST_DATA_CT_PROJECT_KEY = "TEST_DATA_CT_PROJECT_KEY";
+    protected static final String TEST_DATA_CT_CLIENT_ID = "TEST_DATA_CT_CLIENT_ID";
+    protected static final String TEST_DATA_CT_CLIENT_SECRET = "TEST_DATA_CT_CLIENT_SECRET";
+
+    // other TEST_DATA_ variables see in BaseTenant2Fixture
+
+    protected static final Random randomSource = new Random();
+
+    /**
+     * Only for creation of test data
+     */
+    private static BlockingSphereClient ctpClient;
+    private static LoadingCache<String, Type> typeCache;
+
+    private static URL ctPayoneIntegrationBaseUrl;
+
+    /**
+     * maps legible payment names to payment IDs (and vice versa)
```

Comment:
honestly, i'm also a bit confused with this name, but this one is a legacy old name and used in a lot of functions already, thus don't want to rename right now.

But basic idea: when we create payments, we map our custom names (like _Payment 1_, _Payment 2_ etc) to to created `Payment` objects, then later in other tests and assetions we could fetch payments by these custom names.